x/evm/types/errors.go

```diff
@@ -28,8 +28,9 @@ import (
 )
 
 const (
-	codeErrInvalidState      = uint32(iota) + 2 // NOTE: code 1 is reserved for internal errors
-	codeErrExecutionReverted                    // IMPORTANT: Do not move this error as it complies with the JSON-RPC error standard
+	// It is necessary to start from 100 to avoid overlap with Ethermint
+	codeErrInvalidState      = uint32(iota) + 100 // NOTE: code 1 is reserved for internal errors
+	codeErrExecutionReverted                      // IMPORTANT: Do not move this error as it complies with the JSON-RPC error standard
```

Comment:
@facs95 I think the change on line 32 violates the comment `// IMPORTANT: Do not move this error as it complies with the JSON-RPC error standard`

Reply:
RPCs havent been migrated from Ethermint so they still use the original `error.go` file on Ethermint which respects the JSON-RPC standard